invenio_rdm_records/records/jsonschemas/records/record-v6.0.0.json

```diff
@@ -379,6 +379,24 @@
         }
       }
     },
+    "internal_notes": {
```

Comment:
why changing v6?

Reply:
The JSONSchema (and datamodel) is still on v6, though the mapping was bumped because of the incompatible change in the indexing analyzers.

We can have the two being in different versions (although it's a bit confusing). In the end it doesn't affect indexing, since in the Record API class these can be separately specified.

Reply:
Ah sorry! I confused it with the mapping there. Thanks!